### PR TITLE
Add tram-lined title to \maketitle

### PR DESCRIPTION
**Pull request by yjyao**
hot fix for #5. add the "Particulars of Claim" as part of `\maketitle` (thru `\title{PARTICULAR OF CLAIMS}`) with the surrounding tram lines.

@passamo9: did not use your solution suggested in [comment in #5](https://github.com/passamo9/LaTeX_EnglandWales_Templates/pull/5#issuecomment-778680036) because it doesn't work for multi-line titles (for example `\title{SKELETON ARGUMENT FOR THE APPLICANT \\ REQUEST FOR SUMMARY JUDGMENT}`).

### CODE REVIEW

**Comment by passamo9** [comment]
I see, that's very good, thanks for checking.
Would you mind uploading a pdf of what you have uploaded? Thank you

**Comment by yjyao** [comment]
it's already there. if you navigate to the "FIles changed" tab it'll say there's three files being changed, including the `pleading.pdf`. it's a binary file which cannot be diffed so it's easy to overlook in the diff section. i think you have to click on the dot-dot-dot icon next to the filepath and click "View file."

**Comment by passamo9** [comment]
I'll have to catch up on this tomorrow. Thank you so much for all of your work

**Review by Giebisch — APPROVED** [review]

**Comment by yjyao** [comment]
@passamo9 feel free to merge.

**Comment by yjyao** [comment]
hold off of the merge.

**Comment by passamo9** [comment]
@yjyao Did I fuck something up by merging?

**Comment by yjyao** [comment]
nah all is good. this is what i was referring to earlier: keeping the pleading.pdf in the main branch will result in conflicts in almost all PRs once another is merged. going forward the new PRs won't have to worry about this.

**Comment by passamo9** [comment]
>going forward the new PRs won't have to worry about this.

Why is this? Sorry if I'm pestering you, I really want to learn more about GitHub.

**Comment by yjyao** [comment]
no problem.

first, why the conflicts? PDFs are binary files and cannot be (semantically) diffed. like you can diff a bunch of 0s and 1s but that wouldn't be helpful to us humans. so when two PRs update pleading.pdf, there's gotta be a conflict. 

why #18 resolves this? because it removes the PDFs from the main branch (the PDFs are compiled by CI and committed to a separate branch), so new PRs won't have the PDFs to begin with. no PDFs, naturally no conflicts in PDFs.

**Comment by passamo9** [comment]
Oh I see—is that the example folder in the Code? That makes a lot of sense why you had suggested that.

**Comment by passamo9** [comment]
@Andonome

Can you add to the GDPR a title page with the tramlines and the party names?

image

I can create a separate issue or PR on this if it's easier.

**Comment by yjyao** [comment]
what's GDPR? and yes please create a separate issue. our conversations are all over the place and it's hard to trace.

**Comment by yjyao** [comment]
also the PR is ready for merge.

**Comment by Andonome** [comment]
@yjyao 'GDPR' is the general data protection regulation.

Instead of dealing with data laws in France and the UK, it allows parties to only deal with one general law (the GDPR), which simplifies matters.

It also grants new powers to disciplinary bodies, so fines may be issued equal to up to 4% of a company's global annual income, or €20M (or was it €10? unsure), depending on which is greater.

**Comment by yjyao** [comment]
TIL. thanks.

**Comment by passamo9** [comment]
@Andonome that's correct. It's €10 and €20 depending on the severity of the breach (former 'inadvertent', latter grossly negligent or if sensitive data (e.g. health or criminal convictions) is breached).

**Pull request merged**
